src/cloud-api-adaptor/gcp/README.md

```diff
@@ -101,6 +101,14 @@ gcloud container clusters create my-cluster \
   --num-nodes 3
 ```
 
+> [!NOTE]
+> Starting with GKE version 1.27, GCP configures containerd with the
+> `discard_unpacked_layers=true` flag to optimize disk usage by removing
+> compressed image layers after they are unpacked. However, this can cause
+> issues with PeerPods, as the workload may fail to locate required layers. To
+> avoid this, disable the `discard_unpacked_layers` setting in the containerd
+> configuration.
+
```

Comment:
Nice find @beraldoleal.
Do you mind adding a mention about the discard_unpacked_layers in nydus troubleshooting doc as well - https://github.com/confidential-containers/cloud-api-adaptor/blob/main/src/cloud-api-adaptor/docs/troubleshooting/nydus-snapshotter.md

Reply:
Done.